utils/src/test/scala/com/salesforce/op/utils/json/JsonUtilsTest.scala

```diff
@@ -130,3 +162,12 @@ case class TestDouble
   map: Map[Int, Seq[Long]],
   nested: Option[TestDouble]
 )
+
+case class Person(name: String, address: Address)
+
+case class Address(city: String, state: String)
+
+private object Format extends Enumeration {
```

Comment:
No need to reate enum. Simple create two check function: checkYaml and checkJson

Reply:
Done!